records/scrum_team_charter.md

```diff
@@ -0,0 +1,87 @@
+# Visual Meaning Scrum Team Charter
+
+## Things we always do
+
+- Try to write the best code

```

Comment:
very often people are not happy from the things that they wrote as a code but it is according to requirements or design or limitation in knowledge. How can we say that we try. We assume the best intent so this should be true. If it is true why it is here?